BLSQ/iaso
# IA-3215 fixes instance xml to flat json and question_by_path

Explain what problem this PR is resolving

Related JIRA tickets : IA-3215

## Self proofreading checklist


## Changes

- removed the problematic code
- added a test for by_path and by_name questions
- added a command to fix "past" instances 

## How to test

```
docker compose run --rm iaso manage seed_test_data --mode=seed --dhis2version=2.40.4.1
docker-compose run --rm iaso manage fix_submissions_select_multiple
```

you should see that only the `Community Verification Satisfaction` will get fixed.


## Notes

I've used the `query_set.all().iterator()`. I think it's "memory" safe (won't load 100000 instances in the same loop)
